doc/sphinx/language/coq-library.rst

```diff
@@ -105,17 +105,17 @@ subclass :token:`form` of the syntactic class :token:`term`. The syntax of
 
 .. productionlist::
    form :   True                                           (True)
-        : | False                                          (False)
-        : | ~ `form`                                         (not)
-        : | `form` /\ `form`                                   (and)
-        : | `form` \/ `form`                                   (or)
-        : | `form` -> `form`                                   (primitive implication)
-        : | `form` <-> `form`                                  (iff)
-        : | forall `ident` : `type`, `form`                      (primitive for all)
-        : | exists `ident` [: `specif`], `form`                  (ex)
-        : | exists2 `ident` [: `specif`], `form` & `form`          (ex2)
-        : | `term` = `term`                                    (eq)
-        : | `term` = `term` :> `specif`                          (eq)
+        : False                                          (False)
```

Comment:
There is now a misalignement that would be fixed by removing the additional spacing before `True` on the first line. This is the only place where I've noticed this.